### PR TITLE
Remove feature flags

### PR DESCRIPTION
**Pull request by masvelio**
## Link to Issue
Closes: #8606 
Closes: #8604 

## Description of Changes
- removed `allowlist` and `communityHomepage` feature flags

## Test Plan
- make sure that allowlist works correctly
- community homepage is deleted so should not be available 

## Deployment Plan

1. n/a

### CODE REVIEW

**Review by Israellund — APPROVED** [review]

**Review by kurtisassad — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss [hunk 1]
```diff
@@ -6,32 +6,6 @@
   gap: 24px;
   width: 100%;
 
-  .empty-groups-container {
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss:9]: whie remove 🤔?

**Review by mzparacha — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss (hunk 1 above)
Comment (masvelio) [reply packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss:9]: according to [this ](https://github.com/hicommonwealth/commonwealth/pull/8622/files#diff-923f2c6c210470036b809eb402e17666c58d77a953b2e5529ac382f47c5ca967L50), it shouldn't exist when the allowlist is available. I will double check with @kurtisassad

Comment (kurtisassad) [reply packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss:9]: Yeah basically in the figma the empty groups banner was removed.

Comment (masvelio) [reply packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupsSection.scss:9]: thank you Kurtis!

**Pull request merged**